lib/iris/tests/test_cdm.py

```diff
@@ -1135,7 +1135,13 @@ def assert_is_not_lazy(self, cube):
         self.assertFalse(cube.has_lazy_data())
 
     def test_slices(self):
-        lat_cube = next(self.cube.slices(["grid_latitude",]))
+        lat_cube = next(
+            self.cube.slices(
+                [
+                    "grid_latitude",
+                ]
+            )
+        )
```

Comment:
This formatting looks really horrible, is less readable and seems gratuitous. Is there a configuration we can use to make Black less zealous?